cargo-concordium/src/build.rs

```diff
@@ -216,6 +216,10 @@ pub fn build_contract(
     };
 
     let total_module_len = output_bytes.len();
+    if let Some(out_dir) = out_filename.parent() {
+        fs::create_dir_all(out_dir)
+            .context("Unable to create directory for the resulting smart contract module")?;
```

Comment:
Is there a reason why these messages don't end with a `.`?

Reply:
The other messages we have do.

Reply:
Will fix